src/components/standalone/openvpn_tunnel/CreateOrEditTunnelDrawer.vue

```diff
@@ -232,7 +232,7 @@ async function fetchOptions() {
       ...(await ubusCall('ns.ovpntunnel', 'list-digest')).data.digests.map(
         (digest: { name: string; description: string }) => ({
           id: digest.name,
-          label: digest.name
+          label: digest.name + ' (' + t('standalone.openvpn_tunnel.' + digest.description) + ')'
```

Comment:
```suggestion
          label: digest.name,
          description: '(' + t('standalone.openvpn_tunnel.' + digest.description) + ')'
```
I think the field `description` would be a good fit for this kind of information (other than being consistent with the `auto` field), it would look like this:
![image](https://github.com/NethServer/nethsecurity-ui/assets/9089193/f209fb39-120f-499d-af74-fb66dad13b1b)